pegasus/sites.v3/code.org/views/codebytes_skinnybanner.haml

```diff
@@ -4,11 +4,15 @@
       %img{src:"/images/marketing/CodeBytes_CandyAsset.png", style:"max-width: 95%; padding-left:25px; margin: auto"}
   .col-60{style:"padding:0 20px 0 10px; margin-top: 18px; margin-bottom: 18px;"}
     %p{style:"border: 0; color: #4d575f"}
-      Join us December 7-11 for our CodeBytes live mini-lessons!
+      = I18n.t(:codebytes_skinny_banner_title)
     %p{style:"font-size: 14px; border: 0; color: #4d575f"}
-      Join us each day of CS Education Week for CodeBytes, 20 minute interactive computer science lessons. Streaming live twice daily, CodeBytes is a great way for students to participate in a virtual #HourOfCode!
-    %p{style:"font-size: 14px; border: 0; color: #4d575f"}
-      Learn more at
-      %a{href:"https://code.org/codebytes", target: '_blank', style:"color:#7665a0"}
-        code.org/codebytes
+      = I18n.t(:codebytes_skinny_banner_description)
+      = I18n.t(:codebytes_skinny_banner_english_only)
+    %p{style:"font-size: 14px; border: 0; color: #4d575f", class:"skinny-banner-link"}
```

Comment:
nit: instead of adding the class manually, you can instead do `%p.skinny-banner-link`